Can't install delve, compile error finding sort.Slice

Please answer the following before submitting your issue:

Note: Please include any substantial examples (debug session output,
stacktraces, etc) as linked gists.

1. What version of Delve are you using (`dlv version`)? current master (22af38364b4fe71ba10604cf0f771ed628f0005b as of filling)
2. What version of Go are you using? (`go version`)? 1.7
3. What operating system and processor architecture are you using? Ubuntu 16.04.4 LTS
4. What did you do? Tried to install (via both `go get`, and directly via `make install`)
5. What did you expect to see? A successful installation
6. What did you see instead? 
```
go install -ldflags="-X main.Build=22af38364b4fe71ba10604cf0f771ed628f0005b" github.com/derekparker/delve/cmd/dlv
# github.com/derekparker/delve/pkg/proc
/go/src/github.com/derekparker/delve/pkg/proc/fncall.go:292: undefined: sort.Slice
Makefile:52: recipe for target 'install' failed
make: *** [install] Error 2
```

--- a/Documentation/installation/README.md
+++ b/Documentation/installation/README.md
@@ -1,6 +1,8 @@
 # Installation
 
-Directions for installing Delve on all supported platforms is provided here. Please note you *must* have Go 1.7 or higher installed.
+Directions for installing Delve on all supported platforms is provided here.
+
+Please note you *must* have **Go 1.8** or higher installed in order to compile Delve.
 
 - [OSX](osx/install.md)
 - [Linux](linux/install.md)

